sigar_windows.go

```diff
@@ -183,28 +195,149 @@ func (self *DiskList) Get() error {
 	return nil
 }
 
+func getPidList() ([]C.DWORD, error) {
+	// EnumProcesses doesn't tell you if you've read all the processes,
+	// you need to keep making the array bigger until it isn't full
+	var sizeRead C.DWORD
+	for len := INIT_PID_LIST; len < MAX_PID_LIST; len *= 2 {
+		procList := make([]C.DWORD, len)
+		listSize := C.DWORD(unsafe.Sizeof(procList[0])*uintptr(len))
+		success := C.EnumProcesses(&procList[0], listSize, &sizeRead)
```

Comment:
This whole loop is designed to resize the slice bigger and bigger until `sizeRead` is less than `listSize` - see line 210.